src/org/ods/component/ScanWithAquaStage.groovy

```diff
@@ -428,6 +429,16 @@ class ScanWithAquaStage extends Stage {
     private String computeScannedBranch() {
         def scannedBranch = context.getGitBranch()
         if (scannedBranch.toLowerCase().startsWith("release/")) { // We scanned the default integration branch
+            Map branchesResponse = bitbucket.findRepoBranchesStartingWith(context.getRepoName(), scannedBranch)
```

Comment:
Add a comment to indicate that this is to check if the branch has been created in bitbucket.

Reply:
Done.